pkgs/development/libraries/amdvlk/default.nix

```diff
@@ -21,13 +21,13 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "amdvlk";
-  version = "2021.Q1.2";
+  version = "2021.Q1.3";
 
   src = fetchRepoProject {
```

Comment:
This should be switched to fetchFromGitHub.

Reply:
It can't. The GitHub repository only contains metadata, `repo` is used to retrieve the actual source repositories from that metadata.

Reply:
`fetchRepoProject` refers to [Google’s `repo` tool](https://gerrit.googlesource.com/git-repo/), which fetches a bunch of git repos according to an xml description from another git repository (which is [here](https://github.com/GPUOpen-Drivers/AMDVLK/blob/master/default.xml) in the amdvlk repository that is referred in this package).